log_downloader_service.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import shutil
+from pathlib import Path
+from loguru import logger
```

Comment:
I prefer using loguru I don't really see a reason to change.

Reply:
The question is _why_ do you prefer to use that? I don't want to force you to choose any solution, but you need to present an advantage for using loguru -- something stronger than just "I prefer".
If you use the built-in logging module you'll only need a few more lines (if you want the log file to rotate so that it won't get too big) and you won't need an external module, which makes the program more portable.

Ultimately, the code is yours, but I would like you to present more compelling arguments for the solution you picked.

Reply:
It's easier to setup, has better defaults, rotation and retention is easier and it's more clear and formatted